docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -43,8 +43,13 @@ depending on the desired :ref:`retry policy <fte-retry-policy>`.
   * Fault-tolerant execution of :ref:`read operations <sql-read-operations>` is
     supported by all connectors.
   * Fault tolerant execution of :ref:`write operations <sql-write-operations>`
-    is only supported by the :doc:`/connector/delta-lake`,
-    :doc:`/connector/hive`, and :doc:`/connector/iceberg`.
+    is only supported by the following connectors:
```

Comment:
```suggestion
    is supported by the following connectors:
```
It might be worth removing the "only" as this list grows.

Reply:
Agreed & changed